### PR TITLE
ign -> gz Header Migration : ign-fuel-tools

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/711

### CODE REVIEW

File: include/gz/fuel_tools/Interface.hh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <string>
+#include "gz/fuel_tools/Export.hh"
+#include "gz/fuel_tools/FuelClient.hh"
+
```
Comment (chapulina) [inline include/gz/fuel_tools/Interface.hh:21]: Interesting, I just noticed that this header is missing a header guard, I'll add it in a separate PR.

**Review by chapulina — APPROVED** [review]
LGTM with 🟢 CI

**Comment by chapulina** [comment]
Oh we're missing `Export.hh`:

```
   /github/workspace/include/gz/fuel_tools/ClientConfig.hh:27:10: fatal error: gz/fuel_tools/Export.hh: No such file or directory
     27 | #include "gz/fuel_tools/Export.hh"
        |          ^~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by methylDragon** [comment]
CI is failing because of some REST server issue I think

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**